.github/scripts/release.py

```diff
@@ -619,8 +640,15 @@ def main():
         logIndentPush()
         rel_kernel = KernelRelease(mGit, args.new_kernel_version, args.kernel_commit, git_ssh=args.use_git_ssh,
                                    git_org=args.git_org, repo_path=args.kernel_repo_path, branch=args.kernel_repo_branch,
-                                   main_br_version=args.new_kernel_main_br_version, do_not_push=args.do_not_push)
-        rel_kernel.autoRelease()
+                                   main_br_version=args.new_kernel_main_br_version, do_not_push=args.do_not_push, do_not_update_src = args.do_not_update_src)
+
+        # Check if the script is only called to update the src files with release tags and not do the
+        # actual release.
+        if args.do_not_update_src == True and args.defer_release == True:
```

Comment:
Why do we need this `defer_release`? Can we say that this script does 2 things - 
1. Update source files.
2. Create release. 
And if you want to do both, you need to invoke it twice.